### DISCUSSION TITLE
About tubular secretion

### DISCUSSION THREAD

**Discussion by AlanL526** [Open-Systems-Pharmacology/Forum#1134] [Questions & Problems]
When the elimination of a drug included tubular secretion, how to set up renal clearance?
Can tubular secretion be reflected by "GFR Fraction"? In this case the "GFR fraction" will >1.

**Comment by msevestre** [comment]
Check out the documentation on how to model TS
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow#metabolism

Look for tubular secretion
